server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/http/JMAPApiRoutes.java

```diff
@@ -102,6 +103,7 @@ private Mono<Void> post(HttpServerRequest request, HttpServerResponse response)
                 .contextWrite(jmapAuthContext(session)))
             .onErrorResume(BadRequestException.class, e -> handleBadRequest(response, LOGGER, e))
             .onErrorResume(UnauthorizedException.class, e -> handleAuthenticationFailure(response, LOGGER, e))
+            .onErrorResume(OverQuotaException.class, e -> handleBadRequest(response, LOGGER, e))
```

Comment:
I'm pretty sure the fact that it fails have nothing related with the syntax of the client...

We should handle this within the dedicated class: SetMessagesCreationProcessor and SetMessagesUpdateProcessor, catch the overQuotaException there and handle it.

Reply:
More line 264 as the update on the mailboxIds can be used to copy messages and thus might trigger overquota exceptions ;-)